### ISSUE TITLE
questions about training process of batch normalize layer

### ISSUE THREAD

**Issue by appleleaves** [yuruntian/L2-Net#5]
there are some statements needed to make sure:

1. while the weighting and bias are fixed at 1 and 0 respectfully,###  the mean and var in it should be learned in the training process. Right?

2.the author state in the paper that the bn layer before DIF is not learned. Then it is the mean and var are calculated every time for different input?

**Comment by yuruntian** [comment]
@appleleaves  for all the bn layers the affine parmameters are fixed at 1 and 0,  but the  mean and var are learned.
